crates/router/src/connector/wise/transformers.rs

```diff
@@ -0,0 +1,566 @@
+#[cfg(feature = "payouts")]
+use api_models::payouts::PayoutMethodData;
+#[cfg(feature = "payouts")]
+use masking::PeekInterface;
+use serde::{Deserialize, Serialize};
+
+type Error = error_stack::Report<errors::ConnectorError>;
+
+#[cfg(feature = "payouts")]
+use crate::{
+    connector::utils::RouterData,
+    types::{storage::enums as storage_enums, transformers::ForeignFrom},
+};
+use crate::{core::errors, types};
+
+pub struct WiseAuthType {
+    pub(super) api_key: String,
+    #[allow(dead_code)]
+    pub(super) profile_id: String,
```

Comment:
This is required, however, when we run the router specific using cargo hack (where payouts service is not included), this throws error saying the field is never used; so this is added to prevent such test cases from failing